manifests/bases/kubernetes-nmstate-operator.clusterserviceversion.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: |-
```

Comment:
We may need to keep this so the automatic instance creation still works. I could be wrong, but I thought this was what enabled that.